### PR TITLE
chore(corev3): fix create and add resource e2e flow

### PR DESCRIPTION
**Pull request by jayzhang**

### CODE REVIEW

**Review by hund030 — APPROVED** [review]

**Review by nliu-ms — APPROVED** [review]

File: packages/fx-core/src/core/index.ts [hunk 1]
```diff
@@ -442,6 +445,9 @@ export class FxCore implements v3.ICore {
           }
         }
       }
+      if (ctx.projectSettings?.solutionSettings) {
```
Comment (HuihuiWu-Microsoft) [inline packages/fx-core/src/core/index.ts:448]: I think we need to check input capability to decide whether it's Azure hosting?

Comment (jayzhang) [reply packages/fx-core/src/core/index.ts:448]: I think in the future, we will ignore whether Azure hosting or not.

File: packages/fx-core/src/plugins/solution/fx-solution/v3/addFeature.ts [hunk 2]
```diff
@@ -22,20 +22,31 @@ import { selectSingleFeatureQuestion } from "../../utils/questions";
...
 }
 
 export async function getQuestionsForAddFeature(
   ctx: v2.Context,
   inputs: v2.InputsWithProjectPath
 ): Promise<Result<QTreeNode | undefined, FxError>> {
+  const node = new QTreeNode({ type: "group" });
   const plugins = getAllFeaturePlugins();
   const featureNode = new QTreeNode(selectSingleFeatureQuestion);
+  if (!ctx.projectSetting.solutionSettings?.programmingLanguage) {
```
Comment (HuihuiWu-Microsoft) [inline packages/fx-core/src/plugins/solution/fx-solution/v3/addFeature.ts:46]: Is this question added twice(both in core and solution)?

File: packages/fx-core/src/core/index.ts (hunk 1 above)
Comment (HuihuiWu-Microsoft) [reply packages/fx-core/src/core/index.ts:448]: Currently hosting type is used in extension to check what UI to show.

**Review by xzf0587 — APPROVED** [review]

File: packages/fx-core/src/plugins/solution/fx-solution/v3/addFeature.ts (hunk 2 above)
Comment (jayzhang) [reply packages/fx-core/src/plugins/solution/fx-solution/v3/addFeature.ts:46]: programmingLanguage is now core-level question, solution will not ask it.

**Pull request merged**